python/triqs_dft_tools/sumk_dft_transport.py

```diff
@@ -859,45 +953,66 @@ def conductivity_and_seebeck(Gamma_w, omega, Om_mesh, SP, directions, beta, meth
 
     # initialization
     A0 = {direction: numpy.full((n_q,), numpy.nan) for direction in directions}
```

Comment:
add some comments here what this variable holds and why A1 and A2 are only valid for optics and not for raman.